Sources/MixpanelType.swift

```diff
@@ -210,6 +210,43 @@ extension Array: MixpanelType {
     }
 }
 
+extension NSArray: MixpanelType {
+    /**
+     Checks if this object has nested object types that Mixpanel supports.
+     */
+    public func isValidNestedTypeAndValue() -> Bool {
+        for element in self {
+            guard let _ = element as? MixpanelType else {
+                return false
+            }
+        }
+        return true
+    }
+
+    public func equals(rhs: MixpanelType) -> Bool {
+        if rhs is [MixpanelType] {
```

Comment:
I have 2 questions here;
- Why did not you choose to use `if let rhs = rhs as? [MixPanelType]` conditional binding?
- There is no check for `isValidNestedTypeAndValue` rhs? Can this cause a problem

Reply:
@palayemre I followed the format of the other types in this file